robot/control/Src/radio/RadioLink.cpp

```diff
@@ -83,5 +86,7 @@ bool RadioLink::receive(KickerCommand& kickerCommand,
     motionCommand.bodyWVel = static_cast<float>(control->bodyW) / rtp::ControlMessage::VELOCITY_SCALE_FACTOR;
     motionCommand.dribbler = control->dribbler;
 
+    radioConnected = radio->isConnected();
```

Comment:
I'm trying to think if it matters if this is at the top of this function or not. Specifically, `radio->isAvailable` and `radio->receive` `if` statements would both return false, `radio->isConnected` is false so it will never reach this line to update `radioConnected`. 

In our current implementation of the `ISM43340`, we only update `connected` once, so it doesn't make a difference, but if future changes resulted in the ISM driver (or any other radio driver) being able to update `connected` on the fly. Loosing connection to the wifi should result in the `connected` variable changing, which will result in `radioConnected` here never updating because it never reaches this line since it is reasonable that `radio->isAvailable` and `radio->receive` both fail their `if` statements.

TLDR: We should move this line to the top of the function above any returns.

Reply:
Thinking about this more, just remove the radioConnected variable and return radio->isConnected in that getter function. That solves all the problems and is the same behavior

Reply:
That's a good way to do it